pkg/instance/crypt.go

```diff
@@ -11,7 +11,7 @@ import (
 
 var iv = []byte{83, 108, 97, 118, 97, 32, 85, 107, 114, 97, 105, 110, 105, 33, 33, 33}
 
-func encryptText(text string, key string) (string, error) {
+func encryptText(key string, text string) (string, error) {
```

Comment:
Decryption would fail but not encryption.

I changed from value/key to key/value because I found that more logic. But what do you suggest making this more safe?

Reply:
@tonsV2 read the provided link 😋

Reply:
You want to implement the helper type solution?